internal/search/client/client.go

```diff
@@ -155,6 +157,36 @@ func (s *searchClient) JobClients() job.RuntimeClients {
 	}
 }
 
+func shouldSanitizeSearch(ctx context.Context, db database.DB) bool {
+	c := conf.Get()
+	if c.ExperimentalFeatures != nil && c.ExperimentalFeatures.SearchSanitization != nil {
+		user, err := actor.FromContext(ctx).User(ctx, db.Users())
+		if err != nil {
+			return true
+		}
+
+		if user.SiteAdmin {
+			return false
+		}
+
+		if len(c.ExperimentalFeatures.SearchSanitization.OrgName) > 0 {
```

Comment:
nit: when I first read this, I thought org name was a list of allowed names. I find this a bit more readable at a glance:

```suggestion
		if c.ExperimentalFeatures.SearchSanitization.OrgName != "" {
```